### PR TITLE
chore: add volunteer TuhinBar

### PR DESCRIPTION
**Pull request by TuhinBar**
Got some issues with the previous PR. So made a new one.

### CODE REVIEW

File: data/users/TuhinBar.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Tuhin Bar",
+  "bio": "Full Stack Developer",
+  "image": "https://avatars.githubusercontent.com/u/85868593?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Full Stack development","MERN"]
```
Comment (codewithvoid) [inline data/users/TuhinBar.json:8]: you can remove this from here. Can mention frameworks and technologies like ReactJS, NodeJS, TailwindCSS etc

Comment (TuhinBar) [reply data/users/TuhinBar.json]: @codewithvoid fixed the issue

File: data/users/TuhinBar.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Tuhin Bar",
+  "bio": "Full Stack Developer",
+  "image": "https://avatars.githubusercontent.com/u/85868593?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["Node js","Express","CSS","React Js","Bootstrap"]
```
Comment (codewithvoid) [inline data/users/TuhinBar.json:8]: Looks good. Just 2 things

- Avoid use of spaces `Node js` -> `NodeJS`. Similarly for others.
- Keep tags max 2-3

Comment (TuhinBar) [reply data/users/TuhinBar.json]: @codewithvoid fixed that in the recent commit.

**Comment by codewithvoid** [comment]
Awesome. Thank you!

**Pull request merged**